ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController_SwiftUI.swift

```diff
@@ -4,105 +4,467 @@
-                                    actionButtonAction: hasActionButtonAction ? {} : nil,
-                                    messageButtonAction: hasMessageAction ? {} : nil)
-                .frame(maxWidth: .infinity, minHeight: 150, alignment: .center)
-
-            ScrollView {
-                Group {
-                    Group {
-                        VStack(spacing: 0) {
-                            Text("Content")
-                                .frame(maxWidth: .infinity, alignment: .leading)
-                                .font(.title)
-                            Divider()
-                        }
+    // MARK: - Table view data source
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return NotificationDemoSection.allCases.count
+    }
 
-                        TextField("Title", text: $title)
-                            .autocapitalization(.none)
-                            .disableAutocorrection(true)
-                            .textFieldStyle(RoundedBorderTextFieldStyle())
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return NotificationDemoSection.allCases[section].rows.count
+    }
 
-                        TextField("Message", text: $message)
-                            .autocapitalization(.none)
-                            .disableAutocorrection(true)
-                            .textFieldStyle(RoundedBorderTextFieldStyle())
+    override func tableView(_ tableView: UITableView, titleForHeaderInSection section: Int) -> String? {
+        return NotificationDemoSection.allCases[section].title
+    }
 
-                        TextField("Action Button Title", text: $actionButtonTitle)
-                            .autocapitalization(.none)
-                            .disableAutocorrection(true)
-                            .textFieldStyle(RoundedBorderTextFieldStyle())
+    override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
+        return false
+    }
 
-                        FluentUIDemoToggle(titleKey: "Set image", isOn: $showImage)
-                    }
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let section = NotificationDemoSection.allCases[indexPath.section]
+        let row = section.rows[indexPath.row]
 
-                    Group {
-                        VStack(spacing: 0) {
-                            Text("Action")
-                                .frame(maxWidth: .infinity, alignment: .leading)
-                                .font(.title)
-                            Divider()
-                        }
-                        FluentUIDemoToggle(titleKey: "Has Action Button Action", isOn: $hasActionButtonAction)
-                        FluentUIDemoToggle(titleKey: "Has Message Action", isOn: $hasMessageAction)
-                    }
+        switch row {
```

Comment:
I'm not sure what the standard practice is on this team, but this switch statement is very large and nested. Would it make sense to break out the content of each of the switch statements into well-named helper methods to condense this a bit? No biggie if that's not the done thing, though :)

Reply:
I just took a look and 7 out of 9 of the demo controllers using DemoTableViewController create their rows this way, so it seems like the standard to me! Can be handled in a separate PR if needed :)

Reply:
Sounds good! I think it would be beneficial for this to be condensed for easier reading, but definitely don't block on it, especially since it would be good to clean up in a number of places.